crates/nu-command/src/math/abs.rs

```diff
@@ -46,6 +49,19 @@ impl Command for MathAbs {
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let head = call.head;
+        if let PipelineData::Value(
+            Value::Range {
+                ref val,
+                internal_span,
+            },
+            ..,
+        ) = input
+        {
+            match &**val {
+                Range::IntRange(range) => ensure_bounded(range.end(), internal_span, head)?,
+                Range::FloatRange(range) => ensure_bounded(range.end(), internal_span, head)?,
+            }
+        }
```

Comment:
For convenience you could lift the boundedness check as a method on the `enum Range`, so you wouldn't have to repeat those matches, but that's no blocker.